### PR TITLE
Merge app setup to develop

### PR DESCRIPTION
**Pull request by eliasecasta**
In this pull request some basic html with an image were added as well as CSS styling and a Readme file explaining how to run the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
-# review-your-project
-This project is to showcase the things learned at the TSE training program.
+# Hello-Microverse
+
+> This project is to showcase the things learned at the TSE training program.
+
+![screenshot](./assets/Screenshot_4.png)
```
Comment (eliasecasta) [inline README.md:5]: This image is broken, fix it!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#title {
+  color: red;
+  margin: 0 auto;
+  text-align: center;
+}
+
+#potato {
```
Comment (eliasecasta) [inline styles.css:6]: Assign meaningful names to your ids/variables, instead of potato try using something like main-paragraph

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  color: green;
+}
+
+#tomato {
```
Comment (eliasecasta) [inline styles.css]: Assign meaningful names to your ids/variables, instead of potato try using something like main-image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      <div id="potato">
+        In this project, the main goal is to review this project by myself and
+        then another TSE will have to re-review my review in order to check that
+        everything is fine or if there are still missing changes that need to be
+        applied before continuing.
+      </div>
+    </div>
+    <div>
+      <img id="tomato" src="assets/review.png" alt="review image" />
+    </div>
```
Comment (eliasecasta) [inline index.html:22]: Do not overuse the ```<div>``` tag, for instance, you are missing a global container for all your content which is the ```<main>``` tag that goes just after the ```<body>``` tag. Also, don't forget to use just 1 ```<h1>``` tag per page, you could wrap your title with it in this case to make use of it. This would be a sample structure that would be correct instead of wrapping everything inside divs:

```html
<section>
	<article>
		<figure>
			<img>
			<figcaption></figcaption>
		</figure>
	</article>
</section>
```

**Review by eliasecasta — COMMENTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello, I am Elias, your reviewer for this PR

Well done so far with your project!

Great job applying Gitflow as requested by the instructions, your commit messages and Readme are easy to understand and descriptive enough!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] You are not using semantic tags according to the instructions for this project, I will leave you a small [HTML semantics article](https://www.w3schools.com/html/html5_semantic_elements.asp) explaining what are them and how to correctly apply them to your project, I hope that it will be helpful so you can apply the knowledge to the changes that you need to make to your project :)
- [ ] Your screenshot at your Readme is broken, please fix it in order to display a professional Readme to anyone that checks it!
- [ ] Bad id namings, please improve them.
- [ ] Use Class instead of ID if you are using multiple selectors in your CSS file.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (oloomoses) [reply README.md:5]: Well done on pointing out the broken image. However, it would be better if you use polite words rather than commanding the developer to fix the issue. I would rephrase this feedback to read: _You really did well to include the project screenshot on the readme. However, the image is not being displayed on the readme. probably because the image path is not correct. Kindly let us fix the image path and ensure that the image is displayed_

File: styles.css (hunk 2 above)
Comment (oloomoses) [reply styles.css:6]: Assigning meaningful names is a best practice and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should assign meaningful names. Kindly refer to my previous comment on how to provide constructive feedback.

File: styles.css (hunk 3 above)
Comment (oloomoses) [reply styles.css]: Assigning meaningful names is a best practice and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should assign meaningful names. Kindly refer to my previous comment on how to provide constructive feedback.

File: index.html (hunk 4 above)
Comment (oloomoses) [reply index.html:22]: You really did well to give a sample structure that would help the developer move forward. However, the use of negative statements like `Do not` and `Don't` sounds to be authoritative. You could try this: _`You have overused the <div> tag and it would be better if you use HTML semantics like <main>, <section>, <aside> and so forth instead. Also, it would be better to use just 1 <h1> tag per page`_

File: README.md (hunk 1 above)
Comment (oloomoses) [inline README.md:5]: Well done on pointing out the broken image. However, it would be better if you use polite words rather than commanding the developer to fix the issue. I would rephrase this feedback to read: _You really did well to include the project screenshot on the readme. However, the image is not being displayed on the readme. probably because the image path is not correct. Kindly let us fix the image path and ensure that the image is displayed_

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
+#title {
+  color: red;
+  text-align: center;
+}
+
+#potato {
```
Comment (oloomoses) [inline styles.css:6]: Assigning meaningful names is a best practice and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should assign meaningful names. Kindly refer to my previous comment on how to provide constructive feedback.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  color: green;
+}
+
+#tomato,
+#title {
```
Comment (oloomoses) [inline styles.css:11]: Assigning meaningful names is a best practice and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should assign meaningful names. Kindly refer to my previous comment on how to provide constructive feedback.

File: index.html (hunk 4 above)
Comment (oloomoses) [inline index.html:22]: You really did well to give a sample structure that would help the developer move forward. However, the use of negative statements like `Do not` and `Don't` sounds to be authoritative. You could try this: _`You have overused the <div> tag and it would be better if you use HTML semantics like <main>, <section>, <aside> and so forth instead. Also, it would be better to use just 1 <h1> tag per page`_

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @e71az  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
